modules/mgmt-power-remediation-baremetal-about.adoc

```diff
@@ -0,0 +1,114 @@
+// Module included in the following assemblies:
+
+// * machine_management/mgmt-power-remediation-baremetal
+
+[id="mgmt-power-remediation-baremetal-about_{context}"]
+= About power-based remediation of bare metal
+In a bare metal cluster, remediation of nodes is critical to ensure the overall health of the cluster. Physically remediating a cluster can be challenging and any delay in putting the machine to a safe or an operational state increases the time the cluster remains in a degraded state, and the risk that subsequent failures might bring the cluster offline. Power-based remediation helps counter such challenges.
+
+Instead of reprovisioning the nodes, power-based remediation uses a power controller to power off an inoperable node. This type of remediation is also called power fencing.
+
+The {product-title} uses the `MachineHealthCheck` controller to detect faulty bare metal nodes by initiating power-cycling. Power-based remediation is fast and reboots unhealthy hosts instead of removing them from the cluster.
+
+Power-based remediation meets the following objectives:
+
+* Allows the recovery of control plane nodes
+* Prevents data loss in hyperconverged environments.
+* Minimizes downtime for stateful applications that run on the unhealthy nodes.
+
+[id="machine-health-checks-bare-metal_{context}"]
+== MachineHealthChecks on bare metal
+
+Machine deletion on bare metal cluster triggers reprovisioning of a bare metal host.
+Usually bare metal reprovisioning is a lengthy process, during which the cluster
+is missing compute resources and applications might be interrupted.
+To change the default remediation process from machine deletion to host power-cycle,
+annotate the `MachineHealthCheck` resource with the
+`machine.openshift.io/remediation-strategy: external-baremetal` annotation.
```

Comment:
Note that this part of the procedure will change in 4.9

Reply:
@aireilly There is a sample YAML file in the topic that shows the use of this annotation